src/core/BUILD.gn

```diff
@@ -449,6 +449,7 @@ openthread_core_files = [
   "crypto/context_size.hpp",
   "crypto/crypto_platform.cpp",
   "crypto/ecdsa.cpp",
+  "crypto/ecdsa_tinycrypt.cpp",
```

Comment:
Add to [Android.mk](https://github.com/openthread/openthread/blob/main/Android.mk) as well.